app/views/admin/auctions/needs_attention/_payment_needed.html.erb

```diff
@@ -1,4 +1,4 @@
-<table class="usa-table-borderless">
+<table class="usa-table-borderless" id="table-payment">
```

Comment:
I'm not in love with adding css for the sole purpose of tests -- any way we can use xpath instead?